recipes-dotnet/coreclr/coreclr_1.1.0.bb

```diff
@@ -16,14 +16,20 @@ SRCREV = "release/${PV}"
 LIC_FILES_CHKSUM = "file://LICENSE.TXT;md5=ff80286dabb97a39584a14a3edd91cf2"
 S = "${WORKDIR}/git"
 
+do_fix_target_name() {
+	sed -i s/arm-linux-gnueabihf/${TARGET_SYS}/g cross/arm/toolchain.cmake
```

Comment:
I've added an `${S}/` here before the cross. Otherwise sed complains it cannot find the file